setup.py

```diff
@@ -28,11 +29,12 @@
     ],
     license='MPLv2',
     name='pyfxa-versions',
-    packages=['pyfxa-versions'],
+    packages=find_packages(),
```

Comment:
I do `find_packages` out of laziness. I'm not sure if it makes a big difference one way or the other.

https://setuptools.readthedocs.io/en/latest/setuptools.html#using-find-packages

Reply:
I have no preference!